terraform/scripts/generate_vpc_peering_test.go

```diff
@@ -0,0 +1,138 @@
+package scripts_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"os/exec"
+	"path/filepath"
+)
+
+var _ = Describe("VPC peering", func() {
+	var files []string
+
+	BeforeEach(func() {
+		files, _ = filepath.Glob("../*.vpc_peering.json")
+	})
+
+	Context("config files", func() {
+		It("have all required values", func() {
+			type PeerConfig struct {
+				Peer_name   string
+				Account_id  string
+				Vpc_id      string
+				Subnet_cidr string
```

Comment:
It might be better to use idiomatic Go naming conventions here (eg `SubnetCIDR` etc), and to add json field tags to make it explicit which fields in the JSON doc are being read. Otherwise we may miss some case typos or similar things.